README.md

```diff
@@ -15,83 +15,198 @@ npm run test
 
 This project works a bit differently than a regular test suite for now. We have Jest installed more as a test runner, but we don't really use assertions for now. We use it more to gather and report numbers.
 
-### FPS
+### FCP
 
-To gather average FPS time, we just use the `FPS` function from `TraceModel`.
-For FPS we don't need to specify any components to look for as it will just grab the SPF for the entire page.
+First Contentful Paint (FCP) measures the total time taken from the beginning of a page load to the point any content is rendered on the screen.

```

Comment:
I'm not sure if it's ok to take whole sentence from other sites.
copyright? I'm not familiar with that area.
How about replacing the detailed description with links to major sites such as web.dev?,

Reply:
Yes, it is better that way. I will fix this.